### ISSUE TITLE
Upgrade to 0.9 doc is not working for old namespaces

### ISSUE THREAD

**Issue by dimm0** [rook/rook#2394]
**Is this a bug report or feature request?**
* Bug Report

**Deviation from expected behavior:**
The doc is saying:

We should start up the new toolbox pod before moving on, and this can be our first test of the namespace environment variables.

Although I'm getting:

```
error: the namespace from the provided object "rook-ceph" does not match the namespace "rook". You must pass '--namespace=rook-ceph' to perform this operation.
```

When running `kubectl -n $ROOK_NAMESPACE create -f toolbox.yaml`

**Expected behavior:**
This works:
```
kubectl -n $ROOK_NAMESPACE delete pod rook-ceph-tools
kubectl -n $ROOK_NAMESPACE create -f toolbox.yaml
```

Please delete the namespace from the toolbox yaml, or change the docs.
Thanks!

**How to reproduce it (minimal and precise):**
Follow the guide with old "rook" namespace

**Comment by galexrt** [comment]
@dimm0 The better part would be to move the `sed` and env var configuration further up, as `sed` is correctly used to change the namespace to the one provided by the user, see https://rook.io/docs/rook/v0.9/ceph-upgrade.html#1-configure-manifests

**Comment by dimm0** [comment]
sed command is much further in docs.. And if we move it up, it's possible somebody won't read one.
Not sure what would be better to do

**Comment by travisn** [comment]
Let's fix this in the 1.0 upgrade guide.

**Comment by BlaineEXE** [comment]
I don't believe the toolbox needs updating from v0.9 to v1.0, so I think this is fixed automatically. #2965 is the core upgrade update PR for v1.0

**Comment by BlaineEXE** [comment]
Fixed in #3198
